content/episodes/113-multimedia-no-linux-part-2.rst

```diff
@@ -0,0 +1,176 @@
+Episódio 113: Multimedia no Linux - Parte 2
+###########################################
+:date: 2017-08-14
+:authors: Og Maciel, Elyézer Rezende, Bruno Rocha
+:category: Podcast
+:podcast: https://archive.org/download/castalio-podcast-113/castalio-podcast-113.mp3
+:tags: danilo j. s. bellini, audiolazy, ardour, audacity, AviX, Reaper, Adobe Audition, 
```

Comment:
Please make all tags lowercase to follow what have been used on the other episodes.